### DISCUSSION TITLE
Often Show Xcode warning

### DISCUSSION THREAD

**Discussion by yoodu** [pointfreeco/swift-perception#107] [General]
### Description

The project runs normally, but the following warning often appears when opening files：`External macro implementation type 'PerceptionMacros.PerceptibleMacro' could not be found for macro 'Perceptible()'`

![Xnip2024-11-05_19-54-18](https://github.com/user-attachments/assets/97870711-a6df-4504-8d00-0c76e23e0155)

### Checklist

- [X] I have determined that this bug is not reproducible using Swift's observation tools. If the bug is reproducible using the `@Observable` macro or another tool from the `Observation` framework, please [file it directly with Apple](https://github.com/apple/swift/issues/new/choose).
- [X] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-perception/issues) or [discussion](https://github.com/pointfreeco/swift-perception/discussions).

### Perception version information

1.3.5

### Xcode version information

Xcode 15.4.0 and 16.0.0

**Comment by mbrandonw** [comment]
Hi @yoodu, do you have custom build configurations in your project other than the default "Debug" and "Release"? If so, this is a known issue in Xcode. We highly recommend to not use custom build configurations as it leads to lots of problems, including this one.

Since this isn't an issue with the library I am going to convert it to a discussion. Please feel free to continue the conversation over there!
